docs/FormatsAndConventions.md

```diff
@@ -32,20 +32,94 @@ Commit messages for non-trivial commits should have a _**body**_ giving details
  
 1. Separate subject from body with a blank line
 1. Wrap the body at 72 characters
-1. Use the body to explain what and why vs. how
-
-Here is an example (adapted from [this commit](https://github.com/CS2103AUG2016-T11-C4/main/pull/2/commits/5c5b12d1a75c4a73a8330bfd05e406233694ffa3)):
- 
- ```
- Group overloaded methods together
- 
- To make it easier to see the different ways a method can be called, 
- we should group them together.
- 
- Note that this is not required strictly by the coding standard.
- ```
+1. Use the body to explain WHAT the commit is about and WHY it was done that way. The reader can refer to the diff to 
+   understand HOW the change was done.
   
-Refer to the article _[How to Write a Git Commit Message](http://chris.beams.io/posts/git-commit/)_ for a more detailed explanation.
+   * Give an explanation for the change(s) that is detailed enough so that the reader can judge if it is a
+     good thing to do, without reading the actual diff to determine how well the code does what the explanation 
+     promises to do.<br>
```

Comment:
Ah I see, I failed to see the benefits pointed out in Point 1(a). Thanks for the explanation.